hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -383,6 +383,11 @@ private OMConfigKeys() {
   public static final String
       OZONE_SNAPSHOT_SST_FILTERING_SERVICE_INTERVAL_DEFAULT = "60s";
 
+  public static final String OZONE_SNAPSHOT_DIR_CHECK_POLL_TIMEOUT =
```

Comment:
nit: 
1. I think OM specific keys are supposed to follow `ozone.om.*`.
2. may be `ozone.om.snapshot.checkpoint.dir.creation.poll.timeout` .

Reply:
done